coalib/output/ConsoleInteraction.py

```diff
@@ -75,7 +75,7 @@ def highlight_text(no_color, text, style, lexer=TextLexer()):
                          'cannot be printed because it refers to a line '
                          "that doesn't seem to exist in the given file.")
 STR_PROJECT_WIDE = 'Project wide:'
-STR_ENTER_NUMBER = 'Enter number (Ctrl-{} to exit): '.format(
+STR_ENTER_LETTER = 'Enter letter (Ctrl-{} to exit): '.format(
```

Comment:
This change is unnecessary. This changes the ouput message we get, when we call coala without the `-A` tag.